static/panes/editor.js

```diff
@@ -1350,7 +1351,7 @@ Editor.prototype.addSource = function (arr, source) {
 };
 
 Editor.prototype.getAllOutputAndErrors = function (result, compilerName, compilerId) {
-    const compilerTitle = compilerName + ' #' + compilerId;
+    var compilerTitle = compilerName + ' #' + compilerId;
```

Comment:
I don't think this gets modified - was this an intentional change or a remnant of experimentation?
```suggestion
    const compilerTitle = compilerName + ' #' + compilerId;
```

Reply:
I did this because #3753 did something similiar. The solution would be to port this all to TS sooner rather than later